apps/condo/domains/billing/schema/BillingAccount.js

```diff
@@ -65,6 +66,21 @@ const BillingAccount = new GQLListSchema('BillingAccount', {
             isRequired: false,
         },
 
+        isClosed: {
+            schemaDoc: 'Shows whether the billing account closed or not',
+            type: 'Checkbox',
+            defaultValue: false,
+            kmigratorOptions: { default: false },
+        },
+
+        ownerType: {
```

Comment:
may be just `type` ?

Reply:
`type` of account? Or `type` of billing?
`ownerType` is clear.